image-streams.yaml

```diff
@@ -0,0 +1,170 @@
+kind: List
+apiVersion: v1
+metadata:
+  name: image-streams
+  annotations:
+    description: ImageStream definition for Red Hat AMQ Streams 1.0
+    openshift.io/provider-display-name: Red Hat, Inc.
+items:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: amqstreams10-clustercontroller-openshift
+    annotations:
+      openshift.io/display-name: AMQ Streams Cluster Controller     
+      openshift.io/provider-display-name: Red Hat, Inc.
+      version: 1.4.11
+  labels:
+    xpaas: 1.4.11
+  spec:
+    tags:
+    - name: '1.0-TP'
```

Comment:
If I remember the specification of `ImageStreams` correctly, this means that the Docker images will loaded into OpenShift under the tag `1.0-TP`, right? I wonder if it wouldn't be better to use just `1.0`. The tag needs to be specified in the AMQ Streams cluster controller configuration - and using different tags might make it more complicated for the documentation.

Reply:
Yes, docker images for every tag we specify in the image-stream will be loaded into OpenShift. So as the image-stream is now, the tech-preview release images would be under the tag 1.0-TP. When we release the 1.0 GA version, we would add a 1.0 tag to the image-stream pointing to the 1.0 GA images. That way, when a users load the image-stream, they will be able to choose between using the 1.0-TP or the 1.0 versions of the images. Here is an example ( https://github.com/jboss-openshift/application-templates/blob/master/amq/amq62-image-stream.json )